### PR TITLE
WIP BXMSDOC-6395: Added a note for Namespace and DMN Model Name fields value to be same

### PR DESCRIPTION
**Pull request by hmanwani-rh**
Added a Note for Namespace and DMN Model Name fields in **10.1 Invoking a Decision Model and Notation (DMN) service in a business process** in the process designer document.

Doc preview:
[Developing Process Services in Red Hat Process Automation Manager](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6395-RHPAM/#dmn-execution-business-process)

### CODE REVIEW

**Review by kaldesai — APPROVED** [review]
LGTM :)

**Comment by inodeman** [comment]
Looks good!

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/dmn-execution-business-process.adoc [hunk 1]
```diff
@@ -57,6 +57,8 @@ image::dmn/dmn-execution-business-process.png[]
 * *Namespace*: Enter the unique namespace from the DMN model file. Example: `\https://www.drools.org/kie-dmn`
 * *Decision Name*: Enter the name of the DMN decision node that you want to invoke in the selected process node. Example: `Rail`
 * *DMN Model Name*: Enter the DMN model name. Example: `Compute Rail`
++
+IMPORTANT: When you explore the root node, ensure that the *Namespace* and *DMN Model Name* fields consists of the same value.
```
Comment (MarianMacik) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/dmn-execution-business-process.adoc]: If they have to be same, then we should also provide same examples, right? As right now it's `https://www.drools.org/kie-dmn` and `Compute Rail`.

Comment (hmanwani-rh) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/dmn-execution-business-process.adoc]: @MarianMacik, thanks for raising this. The idea is that the value of Namespace and DMN Model Name should be same in BPMN as DMN diagram. To make it clearer, how about changing this to:

IMPORTANT: When you explore the root node, ensure that the *Namespace* and *DMN Model Name* fields consist of the same value in BPMN as DMN diagram.

Comment (MarianMacik) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/dmn-execution-business-process.adoc]: I see now, thanks for clarification. Yes, you can go ahead with that note.

Comment (hmanwani-rh) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/dmn-execution-business-process.adoc]: @MarianMacik, Done. Can you please take a look and approve the changes?

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**